o-fish-ios/Views/Components/BoardVesselButton.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  BoardVesselButton.swift
+//  
+//  Created on 7/3/20.
+//  Copyright © 2020 WildAid. All rights reserved.
+//
+
+import SwiftUI
+
+struct BoardVesselButton: View {
+
+    @ObservedObject var onDuty: DutyState
+    var report = ReportViewModel()
+    var onHeader = true
+
+    @State private var showingReportRootView = false
+    @State private var showingGoOnDutyAlert = false
+
+    private enum Dimensions {
+        static let mainSpacing: CGFloat = 24.0
+        static let padding: CGFloat = 16.0
+        static let bottomPadding: CGFloat = 45.0
+    }
+
+    private var prefilledReport: ReportViewModel {
+        let startReport = ReportViewModel()
+        startReport.vessel.name = report.vessel.name
+        startReport.vessel.homePort = report.vessel.homePort
+        startReport.vessel.permitNumber = report.vessel.permitNumber
+        startReport.vessel.nationality = report.vessel.nationality
+        return startReport
+    }
+
+    var body: some View {
+        NavigationLink(destination: ReportNavigationRootView(report: prefilledReport,
```

Comment:
move NavigationLink to view bottom, keep only CallToActionButton here
(i.e. like in ReportList)